### PR TITLE
Imaging loaders should handle directory of FITS files

### PR DESCRIPTION
**Pull request by OwenArnold**
As well as TIFF files currently handled

### CODE REVIEW

File: ess/v20/imaging/helper_funcs.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import scipp as sc
 from tifffile import imsave
```
Comment (nvaytet) [inline ess/v20/imaging/helper_funcs.py]: i wonder why you need both `fabio` and `tifffile`? Can't you read the tiffs also with `tifffile`?

File: ess/v20/imaging/helper_funcs.py [hunk 2]
```diff
@@ -81,7 +97,20 @@ def tiffs_to_variable(tiff_dir, dtype=np.float64, with_variances=True):
     if with_variances:
         return sc.Variable(["t", "spectrum"], values=data, variances=data)
     else:
+        return sc.Variable(["t", "spectrum"], values=data)
+
+
+def fits_to_variable(fits_dir, dtype=np.float64, with_variances=True):
```
Comment (nvaytet) [inline ess/v20/imaging/helper_funcs.py:110]: Remove duplicate code between `tiffs_to_variable` and `fits_to_variable` by making a generic function called by both?

**Review by nvaytet — CHANGES_REQUESTED** [review]

File: ess/v20/imaging/helper_funcs.py (hunk 1 above)
Comment (OwenArnold) [reply ess/v20/imaging/helper_funcs.py]: I think so. There is some history here I don't recall, but I would definitely be in favour of dropping one.

File: ess/v20/imaging/helper_funcs.py (hunk 2 above)
Comment (OwenArnold) [reply ess/v20/imaging/helper_funcs.py:110]: I had thought to do that, I think we should do this.

**Review by nvaytet — APPROVED** [review]

**Pull request merged**